### PR TITLE
Migrations/seeds

### PR DESCRIPTION
**Pull request by alanperez**
# Description

Created the Users table that will be referencing the Posts table.
minor name change in side the user seed data

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [x] Complete, tested, ready to review and merge
- [x] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

Tested Locally and able to query the data
- [x] Test A
- [x] Test B

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My code has been reviewed by at least one peer
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

File: data/migrations/01_problems.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      .string('problem_title')
+      .notNullable();
+    tbl
+      .string('problem_description')
+      .notNullable();
+    tbl
+      .string('problem_category') // if dropdown is it still a string?
+      .notNullable();
+    tbl
+      .integer('date') // for dates is int correct?
```
Comment (sosafarasa) [inline data/migrations/01_problems.js]: I'll update this to .date and still looking into the dropdown category.

File: data/seeds/users.js [hunk 2]
```diff
@@ -1,13 +1,12 @@
...
-        { username: 'TestUser1', email: 'testuser1@email.com' },
-        { username: 'TestUser2', email: 'testuser2@email.com' },
-        { username: 'TestUser3', email: 'testuser3@email.com' }
-      ])
-    })
-}
+      knex('users').insert([
+        { full_name: 'TestUser1', email: 'testuser1@email.com' },
+        { full_name: 'TestUser2', email: 'testuser2@email.com' },
+        { full_name: 'TestUser3', email: 'testuser3@email.com' }
```
Comment (sosafarasa) [inline data/seeds/users.js]: needs to be finished (add problem_id field)

File: data/migrations/01_problems.js (hunk 1 above)
Comment (alanperez) [reply data/migrations/01_problems.js]: 👍

File: data/seeds/users.js (hunk 2 above)
Comment (alanperez) [reply data/seeds/users.js]: 👍  ok

**Pull request merged**